combat-trainer.lic

```diff
@@ -715,6 +715,8 @@ class SpellProcess
       end
     end
 
+    waitrt? while XMLData.prepared_spell == 'Vivisection' && game_state.use_stealth_attack? && ! hide?
```

Comment:
I would not recommend using a `while` here. I prefer a single attempt to hide to avoid getting caught in a loop or spending a lot of time trying to hide if your chance of success is so-so.